docs/release-notes.md

```diff
@@ -0,0 +1,98 @@
+# Release notes
+
+Refer to the [CHANGELOG](https://github.com/contentauth/c2pa-rs/blob/main/CHANGELOG.md) for detailed changes derived from Git commit history.
+
+## New API 
+
+The current release has a new API that replaces the previous methods of reading and writing C2PA data, which are still supported but will be deprecated.  
+
+The new API focuses on streaming I/O and supports the following structs:
+- [Builder](https://docs.rs/c2pa/latest/c2pa/struct.Builder.html)
+- [Reader](https://docs.rs/c2pa/latest/c2pa/struct.Reader.html)
+- [ManifestDefinition](https://docs.rs/c2pa/latest/c2pa/struct.ManifestDefinition.html)
+
+### Goals
+
+The goals of this release are to provide a consistent, flexible, well-tested API; specifically:
+
+- Move toward a JSON + binary resources model that ports well to multiple languages.
+- Eliminate multiple variations of functions for file/memory/stream, sync/async & etc.
+- Have one stream-based version of each function that works sync and async.
+- Design APIs that work well for multiple language bindings.
+- Enable sign-only/verify-only and support usage without OpenSSL.
+- Support Box Hash and Data Hashed signing models.
+- Enable builds for cameras and other embedded environments.
+- Provide a consistent model for setting runtime options.
+- Keep porting as simple as possible.
+
+### Enabling 
+
+<!-- This requirement should go away with actual 1.0 release, right? -->
+
+To use the new API, enable the `unstable_api` feature; for example:
+
+```
+c2pa = {version="0.39.0", features=["unstable_api"]}
+```
+
+When version 1.0 of the library is released, the new API will become the default, but you will still be able to use the deprecated API by enabling the `v1_api` feature; for example:
+
+```
+c2pa = {version="0.39.0", features=["v1_api"]}
+```
+
+## Language binding support
+
+<!-- Not sure where this really belongs... -->
+
+ | Module         | Method                             |  C++ | Python | WASM | Node  |
+ | --------       | ---------------------------------- |----- | ------ | ---- | ----- |
+ | Builder        |                                    |      |        |      |       |
+ |                | new                                |      |        |      |       |          
+ |                | from_json                          |   X  |   X    |   X  |       |
+ |                | set_claim_generator_info           |      |        |      |       |  
+ |                | set_format                         |      |        |      |       | 
+ |                | set_remote_url                     |      |   X    |      |       | 
+ |                | set_no_embed                       |      |   X    |      |       | 
```

Comment:
C++ now supports set_remote_url, set_no_embed, data_hashed_placeholder and sign_data_hashed_embeddable.

Reply:
Actually need to update a bunch of things on this chart, so just submit as is and I'll follow up.